logback-site/src/site/pages/manual/menu.js

```diff
@@ -1,5 +1,15 @@
+(function() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Move the invocation into the parens that contain the function.](https://www.codacy.com/app/tony19/logback/file/2375991541/issues/source?bid=3222618&fileBranchId=3222700#l1)